temporal-spring-boot-autoconfigure-alpha/src/test/resources/pkcs8-crt-chain.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
@Spikhalskiy I copied the certs from temporal-serviceclient. I noticed the certificate is expiring at the end of the year, we should probably update them to avoid unit tests failing in a few months.